python/tk_multi_loader/dialog.py

```diff
@@ -1687,9 +1687,11 @@ def _on_search_text_changed(self, pattern, tree_view, proxy_model):
             tree_view.setStyleSheet(
                 """QTreeView { border-width: 3px;
```

Comment:
you'll need to escape/double up the actual braces here as well
QTreeView **{{**. ... **}}**

Reply:
Fixed